earthpy/tests/test_hist.py

```diff
@@ -0,0 +1,155 @@
+    """Test that a single custom title works for one band hists."""
+    custom_title = ["Great hist", "another title"]
+
+    with pytest.raises(
+        ValueError, match="You have one array to plot but more than one"
+    ):
+        ep.hist(image_array_single_band, title=custom_title)
+    plt.close()
+
+
+def test_title_string(basic_image):
+    """Test that custom titles work for one band hists."""
+    custom_title = "Great hist"
+    f, ax = ep.hist(basic_image, title=custom_title)
+    assert ax.get_title() == custom_title
+    plt.close(f)
+
+
+def test_multiband_hist_title(image_array_2bands):
+    """Test that custom titles work for multiband hists."""
+    custom_titles = ["Title 1", "Title 2"]
+    f, ax = ep.hist(image_array_2bands, title=custom_titles)
+    num_plts = image_array_2bands.shape[0]
+    assert [f.axes[i].get_title() for i in range(num_plts)] == custom_titles
+    plt.close(f)
+
+
+def test_number_of_hist_bins(basic_image):
+    """Test that the number of bins is customizable."""
+    n_bins = [1, 10, 100]
+    for n in n_bins:
+        f, ax = ep.hist(basic_image, bins=n)
+        assert n == len(ax.patches)
+        plt.close()
+
+
+def test_hist_bbox(basic_image):
+    """Test that the bbox dimensions are customizable."""
+    f, ax = ep.hist(basic_image, figsize=(50, 3))
+    bbox = str(f.__dict__.get("bbox_inches"))
+    assert bbox == "Bbox(x0=0.0, y0=0.0, x1=50.0, y1=3.0)"
+    plt.close()
+
+
+def test_hist_color_single_band(basic_image):
+    """Check that the color argument works for single bands."""
+    f, ax = ep.hist(basic_image, colors=["red"])
+    facecolor = ax.patches[0].__dict__.get("_original_facecolor")
+    assert np.array_equal(facecolor, np.array([1.0, 0.0, 0.0, 1.0]))
+    plt.close(f)
+
+
+def test_hist_color_multi_band(image_array_2bands):
+    """Test that multiple colors work for multiband images."""
+    f, ax = ep.hist(image_array_2bands, colors=["red", "blue"])
+    colors = [a.patches[0].__dict__.get("_original_facecolor") for a in ax]
+    expected_colors = [
+        np.array([1.0, 0.0, 0.0, 1.0]),
+        np.array([0.0, 0.0, 1.0, 1.0]),
+    ]
+    for i in range(2):
+        assert np.array_equal(colors[i], expected_colors[i])
+    plt.close(f)
+
+
+def test_hist_number_of_columns(image_array_2bands):
+    """Test that the col argument changes the number of columns."""
+    number_of_columns = [1, 2]
+    for n in number_of_columns:
+        f, ax = ep.hist(image_array_2bands, cols=n)
+        assert [a.numCols for a in ax] == [n] * 2
+        plt.close(f)
+
+
+# not sure how the tests below are different...
+def test_hist_plot_1_band_array(basic_image):
+    f, ax = ep.hist(basic_image)
+    assert len(f.axes) == 1
+    plt.close(f)
+
+
+def test_hist_plot_1_dim(image_array_2bands):
+    array_1_dim = image_array_2bands.ravel()
+    f, ax = ep.hist(array_1_dim)
+    assert len(f.axes) == 1
+    plt.close(f)
+
+
+# TODO: test that adding an x and y label to the plot works
```

Comment:
@nkorinek would you like to work on writing two more tests? we should test that the x and y labels actually work. note that i didn't implement this on a multi band plot just a single so far. you are welcome to add it to multiband if you'd like!! 

below we need a test that ensures a color provided as a string works! you may need to add a test to the hist function as well to coerce a color provided as a string to a list just like i did with the title.